io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -875,4 +875,33 @@ impl SnapshotOps for Lvol {
     async fn snapshot_iter(self) -> LvolSnapshotIter {
         LvolSnapshotIter::new(self)
     }
+    /// Prepare Snapshot Config for Block/Nvmf Device, before snapshot create.
+    async fn prepare_snap_config(
+        &self,
+        snap_name: &str,
```

Comment:
IIRC should the txn_id be passed as argument as the control plane would want to set it?

Reply:
updated code. txn_id will be part of gRPC request.

Reply:
Yes, tx_id should be a mandatory gRPC argument.